### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by dobooth**
## This PR merges:

- #4177
- #4203
- #4195 
- #4193 
- #4181 
- #4178 
- #4191 
- #4206 
- #4207 
- #4208

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4204) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4204) before we can accept your contribution.<br/>**6** out of **7** committers have signed the CLA.<br/><br/>:white_check_mark: manthandave<br/>:white_check_mark: likemusic<br/>:white_check_mark: dobooth<br/>:white_check_mark: hiteshvaghasiya83<br/>:white_check_mark: erikmarr<br/>:white_check_mark: hguthrie<br/>:x: ImgBotApp

File: guides/v2.1/install-gde/composer.md [hunk 1]
```diff
@@ -41,19 +41,19 @@ To get the Magento metapackage:
 
     **{{site.data.var.ce}}**
     ```bash
-    composer create-project --repository=https://repo.magento.com/magento/project-community-edition:<version-tag> <install-directory-name>
+    composer create-project --repository=https://repo.magento.com/ magento/project-community-edition:<version-tag> <install-directory-name>
```
Comment (erikmarr) [inline guides/v2.1/install-gde/composer.md]: Looks like an extra space was added.

File: guides/v2.1/install-gde/composer.md [hunk 2]
```diff
@@ -41,19 +41,19 @@ To get the Magento metapackage:
...
     ```
 
     **{{site.data.var.ee}}**
     ```bash
-    composer create-project --repository=https://repo.magento.com/magento/project-enterprise-edition:<version-tag> <install-directory-name>
+    composer create-project --repository=https://repo.magento.com/ magento/project-enterprise-edition:<version-tag> <install-directory-name>
```
Comment (erikmarr) [inline guides/v2.1/install-gde/composer.md]: Looks like an extra space was added.

File: guides/v2.1/install-gde/composer.md [hunk 3]
```diff
@@ -41,19 +41,19 @@ To get the Magento metapackage:
...
     ```
 
     When prompted, enter your Magento authentication keys. Your _public key_ is your username; your _private key_ is your password.
 
     To install {{site.data.var.ce}} version 2.2.2 run in current folder:
     ```bash
-    composer create-project --repository=https://repo.magento.com/magento/project-community-edition:2.2.2 .
+    composer create-project --repository=https://repo.magento.com/ magento/project-community-edition:2.2.2 .
```
Comment (erikmarr) [inline guides/v2.1/install-gde/composer.md]: Looks like an extra space was added.

File: guides/v2.1/install-gde/composer.md (hunk 1 above)
Comment (dobooth) [reply guides/v2.1/install-gde/composer.md]: That's the fix. I was surprised myself.

File: guides/v2.1/install-gde/composer.md (hunk 2 above)
Comment (dobooth) [reply guides/v2.1/install-gde/composer.md]: See above.

File: guides/v2.1/install-gde/composer.md (hunk 3 above)
Comment (dobooth) [reply guides/v2.1/install-gde/composer.md]: Same.

**Comment by dobooth** [comment]
running tests

**Pull request merged**